lib/__tests__/standalone-parseErrors.test.mjs

```diff
@@ -1,37 +1,18 @@
-import { createRequire } from 'node:module';
-import { jest } from '@jest/globals';
-
-import readJSONFile from '../testUtils/readJSONFile.mjs';
 import replaceBackslashes from '../testUtils/replaceBackslashes.mjs';
 import standalone from '../standalone.mjs';
 
-const configBlockNoEmpty = readJSONFile(
-	new URL('./fixtures/config-block-no-empty.json', import.meta.url),
-);
-
-jest.mock('../rules/block-no-empty/index.cjs');
-
-const require = createRequire(import.meta.url);
-const blockNoEmpty = require('../rules/block-no-empty/index.cjs');
-
-blockNoEmpty.mockImplementation(() => {
-	return (root, result) => {
-		result.warn('Some parseError', {
-			stylelintType: 'parseError',
-		});
-	};
-});
-
 test('standalone with deprecations', async () => {
 	const data = await standalone({
-		code: 'a {}',
-		config: configBlockNoEmpty,
+		code: 'a[=] {}',
+		config: { rules: { 'selector-attribute-quotes': 'always' } },
 	});
 
-	expect(data.output).toContain('Some parseError');
+	expect(data.output).toContain('"stylelintType":"parseError"');
```

Comment:
question

It should be `report` not `output`, correct?

Reply:
Good catch. Fixed with 2d89bf201. 👍🏼